frontend/src/views/PublicCanteenStatisticsPage/index.vue

```diff
@@ -0,0 +1,379 @@
+<template>
+  <div class="text-left grey--text text--darken-4">
+    <h1 class="text-h4 font-weight-black black--text mt-3 mb-6">Découvrir les démarches chez vous</h1>
+    <!-- Add some introductory text? -->
+    <v-form class="my-4 pb-8">
+      <v-row>
+        <v-col cols="12" sm="6" md="4">
+          <label for="select-region" class="text-body-2">
+            Région
+          </label>
+          <v-autocomplete
+            v-model="chosenRegion"
+            :items="regions"
+            clearable
+            hide-details
+            id="select-region"
+            placeholder="Toutes les régions"
+            class="mt-1"
+            outlined
+            dense
+            auto-select-first
+            :filter="locationFilter"
+          ></v-autocomplete>
+        </v-col>
+        <v-col cols="12" sm="6" md="4">
+          <label for="select-department" class="text-body-2">
+            Département
+          </label>
+          <v-autocomplete
+            v-model="chosenDepartment"
+            :items="departments"
+            clearable
+            hide-details
+            id="select-department"
+            placeholder="Tous les départements"
+            class="mt-1"
+            outlined
+            dense
+            auto-select-first
+            :filter="locationFilter"
+          ></v-autocomplete>
+        </v-col>
+        <v-col cols="12" sm="6" md="4" class="d-flex align-end">
+          <v-btn x-large color="primary" @click="submit">
+            <!-- More descriptive text? -->
+            Rechercher
+          </v-btn>
+        </v-col>
+      </v-row>
+    </v-form>
+    <div v-if="locationText" class="py-8">
+      <h2 class="text--darken-5 text-h4 mb-8">Les statistiques pour {{ locationText }}</h2>
```

Comment:
Petite suggestion : Dans la plupart des pages on joue sur l’épaisseur du texte pour les titres. On peut le faire pour cette page aussi si ça te va. Du coup on n'aurait pas besoin du `text--darken-5`.
Si on prend cette voie, ce titre serait un `text-h5` et les autres un `text-h6` (mais avec le `font-weight-black` ça prend à peu près la même dimension).
```suggestion
      <h2 class="text-h5 font-weight-black mb-8">Les statistiques pour {{ locationText }}</h2>
```